.github/workflows/secret-rotation-check.yml

```
@@ -0,0 +1,444 @@
+
+            // Check for existing open issue
+            const { data: issues } = await github.rest.issues.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              labels: 'secret-rotation'
+            });
+
+            const body = `## Secret Rotation Required
+
+The weekly secret rotation check has identified secrets that need attention.
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)